### ISSUE TITLE
Make it official?

### ISSUE THREAD

**Issue by jeffam** [a-fro/drupal_pullquotes#1]
I wonder if this would be a good candidate for the D8 version of the existing Pullquote module: https://www.drupal.org/project/pullquote

**Comment by a-fro** [comment]
Yeah, good point. Maybe you know someone with contrib rights on D.O?  ;) 

Based on my current workload, it's not likely to be me.
